tests/test_component.py

```diff
@@ -23,13 +24,14 @@
 from django.test import Client
 from django.urls import path
 from django.utils.safestring import SafeString
+from pytest_django.asserts import assertHTMLEqual, assertInHTML
```

Comment:
So I mentioned in one of the other comments that I moved the assertion from Django's assertion helpers to plain assertion statements. However, there's two assertions `assertHTMLEqual` and `assertInHTML`, which have very specific behaviour, and so I kept them.

But to keep them while still using pytest for collecting tests, I had to add `pytest-django`, which also adds [all the Django's assertions](https://pytest-django.readthedocs.io/en/latest/helpers.html#assertions).

So as I've noted in https://github.com/django-components/django-components/issues/1004, once we get to working on decoupling django-components from Django, these assertions will have to be replaced. But until then it's fine.